### DISCUSSION TITLE
I should not be rejected

### DISCUSSION THREAD

**Discussion by ZZWaang** [community/community#132138] [GitHub Education]
### Hi everyone,

I'm clearly a student based on what I have uploaded. Why keep rejecting me?

**Comment by davevad93** [comment]
Hi @ZZWaang ,

Take a look at the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352), there you'll find info about common rejection reasons and how to address them.

**Comment by ZZWaang** [comment]
I've checked and applied multiple times. Please help me with it. Thank you.

**Reply by davevad93** [reply]
I'm sorry @ZZWaang I can't help you with that, I'm just a standard user. Hopefully someone who's in charge of GitHub Education will see this discussion and will be able to help you.
